RegExp.make.js

```diff
@@ -37,7 +37,7 @@ RegExp.make = (function () {
     if (0x20 <= n && n <= 0x7e) {
       return String.fromCharCode(n).replace(UNSAFE_CHARS_CHARSET, '\\$&');
     }
-    var hex = n.toString(16);
+    const hex = n.toString(16);
```

Comment:
check.sh complains

```
egExp.make.js:40: ERROR - Unable to infer type of constant.
    const hex = n.toString(16);
          ^

1 error(s), 0 warning(s)
```

with the const.

Adding an explicit

```javascript
/** @type {string} */
```

makes it go away, and I find closure compiler catches useful things often enough that I'd like to keep it running clean.